windows/src/desktop/kmshell/main/OnlineUpdateCheck.pas

```diff
@@ -49,7 +51,9 @@ interface
 type
   EOnlineUpdateCheck = class(Exception);
 
-  TOnlineUpdateCheckResult = (oucUnknown, oucShutDown, oucSuccess, oucNoUpdates, oucFailure, oucOffline);
+  TOnlineUpdateCheckResult = (oucUnknown, oucShutDown, oucSuccess, oucNoUpdates, oucUpdatesAvailable, oucFailure, oucOffline);
+
+  TUpdateState = (idle, check, download, pending, installing, postinstall);
```

Comment:
For Delphi enums we normally use a shorthand prefix. It's not critical but it is consistent with the rest of the code:

```suggestion
  TUpdateState = (usIdle, usCheck, usDownload, usPending, usInstalling, usPostInstall);
```